runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -164,6 +164,12 @@
         @Default
         long maxCacheWeight;
 
+        /**
+         * A parameter to tune memory usage vs cache concurrency
+         */
+        @Default
+        int cacheConcurrency = 0;
```

Comment:
It would be better if more comments about what cacheConcurrency means.

Reply:
I agree, without some comments on what this means it is highly possible consumers can end up misusing it.

Reply:
+1 can we add some more info. Perhaps some hint on how to set? number of threads that could potentially access the cache at one time? not sure.

Reply:
Done.